sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/patch/implementation/PatchOperationType.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.patch.implementation;
+
+public enum PatchOperationType {
```

Comment:
Needs to be public?

Reply:
It's needed as I am accessing it in another package. But this is in implementation package. So should be fine.